provider/resources.go

```diff
@@ -3094,6 +3094,7 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 				},
 				TransformFromState: func(ctx context.Context, state resource.PropertyMap) (resource.PropertyMap, error) {
 					if _, ok := state["dbName"]; ok {
+						delete(state, "name")
```

Comment:
It seems that under PlanResourceChange, having "name" hanging around confuses the diff to think it's changing from "something" to "" and that should be a replacement.